imports/plugins/included/payments-stripe/client/checkout/components/injectedCardForm.js

```diff
@@ -0,0 +1,273 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  injectStripe,
+  CardNumberElement,
+  CardExpiryElement,
+  CardCVCElement,
+  PostalCodeElement
+} from "react-stripe-elements";
+import { Meteor } from "meteor/meteor";
+import { i18next, Router } from "client/api";
+
+class CardForm extends Component {
+  static propTypes = {
+    cartId: PropTypes.string,
+    postal: PropTypes.string,
+    stripe: PropTypes.object
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      cardNumberErrorMessage: "",
+      CVVErrorMessage: "",
+      errorMessage: "",
+      expDateErrorMessage: "",
+      postal: props.postal,
+      postalErrorMessage: "",
+      submitMessage: "Complete your order",
+      submitting: false
+    };
+
+    /* eslint-disable camelcase */
+    this.errorCodes = {
+      card_declined: i18next.t("checkout.errorMessages.cardDeclined", "Your card has been declined. Please use another card."),
```

Comment:
Can this error message be retrieved from the package translation file? this will remove code duplication.